src/CqlBuilderPanel/definitionsSection/DefinitionSectionNavTabs.tsx

```diff
@@ -4,10 +4,11 @@ import { Tabs, Tab } from "@madie/madie-design-system/dist/react";
 export interface NavTabProps {
   activeTab: string;
   setActiveTab: (value: string) => void;
+  definitionCount: number;
 }
 
 export default function DefinitionSectionNavTabs(props: NavTabProps) {
-  const { activeTab, setActiveTab } = props;
+  const { activeTab, setActiveTab, definitionCount } = props;
```

Comment:
I think this should be definitionsCount -- plural as we would be dealing with many definitions